public/styles.css

```
@@ -306,10 +306,7 @@
 .terms-and-conditions {
   max-width: 1300px;
   margin: 0 auto;
-  padding-top: 34px;
-  padding-bottom: 30px;
-  padding-left: 48px;
-  padding-right: 48px;
+  padding: 12px 12px 10px 10px;
```

Why is it 12px 12px 10px 10px?

If you are trying to set `top` & `bottom` to `12px` and `left` & `right` to `10px` this is not right. The order is `top right bottom left` (starting with the top and going clockwise).

If the opposite sides are the same, ie top == bottom & left == right, you can just set it by passing two values like this
```suggestion
  padding: 12px 10px;
```